src/sentry/runner/commands/configoptions.py

```diff
@@ -273,10 +273,9 @@ def sync(ctx: click.Context) -> None:
                                 )
                                 presenter_delegator.flush()
```

Comment:
above on line 244, I believe we properly check if an option is already set on disk before applying 

https://github.com/getsentry/sentry/blob/fix/options-on-disk/src/sentry/options/manager.py#L188-L200